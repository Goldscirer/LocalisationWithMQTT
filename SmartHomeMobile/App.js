import React, { Component } from 'react';
import { View, Text } from 'react-native';
import {Button, Header} from './src/Components/common';
import { ScrollView } from 'react-native';

class App extends Component {
  render() {
    return (
        <View>
          <Header headerText={'SMART HOME'} />
            <ScrollView>
                <Button/>
            </ScrollView>
        </View>
    );
  }
}

export default App;
