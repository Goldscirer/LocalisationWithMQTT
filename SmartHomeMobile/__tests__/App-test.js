/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const snap = renderer.create(
      <App/>
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
