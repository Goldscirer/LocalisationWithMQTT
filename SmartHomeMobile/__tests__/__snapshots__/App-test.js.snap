// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#003478",
        "elevation": 10,
        "height": 80,
        "justifyContent": "center",
        "paddingTop": 5,
        "position": "relative",
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 40,
          "width": 0,
        },
        "shadowOpacity": 0.5,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      SMART HOME
    </Text>
  </View>
  <RCTScrollView>
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignSelf": "stretch",
            "backgroundColor": "#fff",
            "borderColor": "#007aff",
            "borderRadius": 5,
            "borderWidth": 1,
            "flex": 1,
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 10,
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "alignSelf": "center",
              "color": "#007aff",
              "fontSize": 16,
              "fontWeight": "600",
              "paddingBottom": 10,
              "paddingTop": 10,
            }
          }
        >
          SEND YOURS LOCATION
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
